common/interactivetx.c

```diff
@@ -83,6 +80,11 @@ static bool is_segwit_output(const tal_t *ctx,
 			     struct wally_tx_output *output,
 			     const u8 *redeemscript)
 {
+	/* Since redeemscript was removed from the spce, we assume it is
+	 * segwit without checking for now. */
+	if (!redeemscript)
+		return true;
```

Comment:
we'll get to this answer eventually, i would remove this.

Reply:
it's outttaa here